### DISCUSSION TITLE
【求助】我进空洞了！

### DISCUSSION THREAD

**Discussion by Drs1145** [share121/inter-knot#1119] [其他]
事情是这样的，我仍然在街上闲逛，听说有一个录像店口碑不错想提前去踩踩点结果走到半路的时候一个伴生空洞忽然出现把我吞下去了！也不知道是哪个街区。第一次遇到这种情况也不知道怎么办于是就想到绳网求助了，也不知道绳匠们平常都要什么东西但是听说菲林这东西挺抢手的，虽然我不知道这是什么但是也算是颇有家资。我在此发誓——谁第一个将我救出去我就给他300菲林还有一些丁尼，这是我的全部家当了，不够还可以商量。
谁都可以，只要把我带出去就行！

**Comment by Drs1145** [comment]
我后悔刚才走路看手机……

**Comment by Drs1145** [comment]
法厄同，法厄楠同，法厄铝同，法厄傻同……都可以，有人这么晚还在吗？

**Reply by sekiroissekiko** [reply]
法厄楠同23333

**Comment by DonQuixote084** [comment]
空洞里还能上绳网？你怕不是最近把绳匠骗到指定地点然后抢劫的盗洞客吧？@六分街治安局 来业绩了

**Comment by Drs1145** [comment]
纳尼？难道我进的不是空洞？

**Comment by Hscp-hscp** [comment]
这个委托我接了

**Comment by Phang03** [comment]
v50菲林看看实力 😆
